packages/api/test/core/prepareParams.test.ts

```diff
@@ -101,6 +101,24 @@ describe('#prepareParams', function () {
     });
   });
 
+  it('should ignore a supplied second parameter if its an empty object', async function () {
+    const spec = await import('@readme/oas-examples/3.0/json/petstore.json').then(Oas.init);
+    await spec.dereference();
+
+    const operation = spec.operation('/pet/findByStatus', 'get');
+
+    const body = {
+      status: ['available'],
+    };
+    const metadata = {};
+
+    expect(await prepareParams(operation, body, metadata)).to.deep.equal({
```

Comment:
Which item here is supposed to be an empty object? from the name of the test I would assume `body` but it's not empty.

Reply:
@Dashron I've updated this to be `await prepareParams(operation, metadata, {})` to hopefully clear up any confusion here.